airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/json_conversion_test_cases.json

```
@@ -778,5 +778,48 @@
       "time_field": 44581541000,
       "_airbyte_additional_properties": null
     }
+  },
+  {
+    "schemaName": "array_without_items_in_schema",
+    "namespace": "namespace16",
+    "appendAirbyteFields": false,
+    "jsonSchema": {
+      "type": "object",
+      "properties": {
+        "identifier": {
+          "type": "array"
+        }
+      }
+    },
+    "jsonObject": {
+      "identifier": ["151", "152"]
```

Could you make the elements in this array a mixture of types? So that we can test that everything is converted to string. Currently both the elements are already strings, and no conversion is needed or tested.

```suggestion
      "identifier": ["151", 152, true, { "id": 153 }]
```

---

@tuliren done